theme/theme.ts

```diff
@@ -1,51 +1,108 @@
 import { ChakraTheme, DeepPartial } from '@chakra-ui/react'
+import { mode } from '@chakra-ui/theme-tools'
 
 const IronFishTheme: DeepPartial<ChakraTheme> = {
+  components: {
+    Table: {
+      variants: {
+        blocks: props => ({
+          table: {
+            borderCollapse: 'separate',
+            borderSpacing: '0px 16px',
```

Comment:
Where possible I'd like to use `rem` instead of pixels, and we should be working from a 16px grid. I wrote some utilities to aid this in `/styles`.